city_scrapers/spiders/bisnd_bps.py

```diff
@@ -0,0 +1,158 @@
+import html
+import json
+import re
+from datetime import datetime, time
+
+from city_scrapers_core.constants import BOARD, CANCELLED, COMMITTEE, PASSED, TENTATIVE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from lxml import html as lhtml
+from scrapy import Selector
+
+
+class BisndBpsSpider(CityScrapersSpider):
+    name = "bisnd_bps"
+    agency = "Bismarck Public Schools"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.bismarckschools.org/Page/401"]
+
+    """
+    Meetings' time and location is taken from organization's following webpage:
+    https://www.bismarckschools.org/Page/398
+    """
+    meeting_time = time(17, 15)
+    location = {
+        "name": "Tom Baker Meeting Room of the City/County Office Building",
+        "address": "221 N Fifth Street, Bismarck, ND",
+    }
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+        """
```

Comment:
**polish (non-blocking):** I generally like to remove the default helper text in our docstrings or tweak the wording to be a bit more explicit (eg. "Parse meeting items from agency website"). This is not a big deal and our style across the city-scraper codebase is inconsistent.